### PR TITLE
Makes more things deconstructable + a few deconstruction adjustments

### PR DESCRIPTION
**Pull request by MatheusIsNotAvailable**
[FEATURE] [QOL]

## About the PR 
Makes big instruments such as the jukebox, portable material reclaimers, the kitchen monkey spike, drone rechargers and botany plant pots all deconstructable makes it so that when you hit a vending machine with a decon tool or a device analyzer it doesn't hit the machine, also makes it so the decon tool menu welder option looks for a tool with the weld flag instead of a welding tool.

## Why's this needed? 
Deconstructing stuff is cool, taking the risk of a vending machine falling on my head every time I want to decon one is funny but annoying, and the omnitool welder should work on the decon menu.

### CODE REVIEW

**Comment by penny64** [comment]
There's too much comedic value to the machines falling on you for scanning/deconstructing them. It's what nerds should get.

**Comment by Awfullyfishy** [comment]
You should look into making the firelocks deconstructable as well, they tend to be left after any construction work

**Comment by MatheusIsNotAvailable** [comment]
> You should look into making the firelocks deconstructable as well, they tend to be left after any construction work

I plan on doing that in the future yeah, it would just require a little more work because I need to make sure they work right when reconstructed, same with light switches, fire alarms and other things that depend on their area. Expect those things in a PR if this one gets approved

File: code/obj/item/material.dm [hunk 1]
```diff
@@ -763,6 +763,7 @@
 	anchored = 0
 	density = 1
 	event_handler_flags = NO_MOUSEDROP_QOL
+	deconstruct_flags = DECON_SCREWDRIVER | DECON_WRENCH | DECON_CROWBAR
```
Comment (Tarmunora) [inline code/obj/item/material.dm:766]:
```suggestion
```
skip the portable reclaimer decon and this is good

**Review by Tarmunora — CHANGES_REQUESTED** [review]

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**